org.eclipse.jdt.core/model/org/eclipse/jdt/internal/core/JavaModel.java

```diff
@@ -406,6 +413,7 @@ public static boolean isJmod(File file) {
  * JavaModelManager's external file cache.
  */
 static private boolean isExternalFile(IPath path) {
+	System.out.println("isExternalFile: "+path); //$NON-NLS-1$
```

Comment:
Please don't forget to remove for final version

Reply:
oh, yes, thanks!